api/mailer/mailer.js

```diff
@@ -53,4 +53,16 @@ router.post('/validateVerificationEmail', async (req, res) => {
     });
 });
 
+router.post('/getCalendarEvents', async (req, res) => {
+  let calendarID = req.query.calendarID;
```

Comment:
we can also fetch the `primary` calendar if the param is undefined, e.g.
```suggestion
  const calendarID = req.query.calendarID || 'primary';
```